lib/cog/queries/alias.ex

```diff
@@ -30,4 +30,16 @@ defmodule Cog.Queries.Alias do
     select: a
   end
 
+  def user_alias_by_name(name) do
+    from a in UserCommandAlias,
+    where: a.name == ^name,
+    select: a
```

Comment:
I think we're using a new enough version of Ecto to use this expression shorthand:

```
UserCommandAlias |> where(name: name)
```